src/index.html

```diff
@@ -12,6 +12,185 @@
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            name="old"
+            id="old"
+            min="1"
+            max="100"
+            value="12"
+            required
+          >
+        </label>
+        
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            id="dateOfBirth"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          I accept the term of the agreement 
+          <input 
+            type="checkbox" 
+            name="termOfTheAgreement" 
+            id="termOfTheAgreement"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="container">
+        <legend>
+          Registration
+        </legend>
+
+        <label class="form-field">
+          E-mail: 
+          <input 
+            type="email" 
+            name="email" 
+            id="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+
+        <label class="form-field">
+          Password: 
+          <input 
+            type="password" 
+            name="password" 
+            id="password"
+            minlength="5"
+            maxlength="30"
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset>
+        <legend>
+          An interesting fact about you!
+        </legend>
+
+        <div class="form-field"></div>
+        Do you love cats?
+        <label>
+          <input 
+            type="radio" 
+            name="loveCat" 
+            value="yes"
+            required
+          >
+          Yes
+        </label>
+        
+        <label>
+          <input 
+            type="radio" 
+            name="loveCat" 
+            value="no"
+            required
+          >
+          No
+        </label>
+          
+        </div>
+
+        <label class="form-field">
+          What is your favirite color?
+          <input 
+            type="color" 
+            name="color" 
+            id="color"
+            required
+          >
+        </label>
+        
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+            type="time" 
+            name="time" 
+            id="time"
+            required
+          >
+        </label>
+        
+        <label class="form-field">
+          What are your favirite brand of cars?
+          <select name="brand" multiple >
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field">
+          How do you rate our work?
+          <input
+          type="range" 
+          name="rateOurWork" 
+          id="rateOurWork"
+          required
+          >
```

Comment:
Check the code style 
```suggestion
          <input
            type="range" 
            name="rateOurWork" 
            id="rateOurWork"
            required
          >
```